app-modules/assistant/src/Filament/Pages/PersonalAssistant.php

```diff
@@ -842,6 +844,7 @@ protected function setMessage(string $message, AIChatMessageFrom $from, string $
             message_id: $messageId,
             message: $message,
             from: $from,
+            created_at: now()
```

Comment:
```suggestion
            created_at: now(),
```

Reply:
I wonder if we can enable trailing comma enforcement in Pint

Reply:
I would think the `trailing_comma_in_multiline` rule should enforce that since it mentions "parameters list" as a case where multiline expressions must have a trailing comma, but it doesn't seem to 🤷